iphone/Classes/TiConsole.m

```diff
@@ -50,6 +50,27 @@ - (void)timeEnd:(id)label
   [_times removeObjectForKey:label];
 }
 
+- (void)timeLog:(id)args
+{
+  NSString *label = [args objectAtIndex:0];
+  NSArray *logData = [args count] > 1 ? [args subarrayWithRange:NSMakeRange(1, [args count] - 1)] : nil;
+  if (label == nil) {
+    label = @"default";
+  }
+  NSDate *startTime = _times[label];
+  if (startTime == nil) {
+    NSString *logMessage = [NSString stringWithFormat:@"Label \"%@\" does not exist", label];
+    [self logMessage:[logMessage componentsSeparatedByString:@" "] severity:@"warn"];
+    return;
+  }
+  double duration = [startTime timeIntervalSinceNow] * -1000;
```

Comment:
Use `NSTimeInterval` instead of `double`. They are the same, but `NSTimeInterval` is more known for storing time values.